packages/react-flight-dom-webpack/src/__tests__/ReactFlightDOM-test.js

```diff
@@ -136,7 +163,11 @@ describe('ReactFlightDOM', () => {
     }
 
     let {writable, readable} = getTestStream();
-    ReactFlightDOMServer.pipeToNodeWritable(<RootModel />, writable);
+    ReactFlightDOMServer.pipeToNodeWritable(
+      <RootModel />,
```

Comment:
Should we delete these non-Block tests now? What is their use?

Reply:
They're the only ones testing async right now but ideally they should be rewritten to a more idiomatic style.